scipy/ndimage/tests/test_filters.py

```diff
@@ -785,26 +770,37 @@ def test_filter_invalid_axes(self, axes, filter_func, kwargs):
             error_class = ValueError
             match = "out of range"
         with pytest.raises(error_class, match=match):
-            filter_func(array, axes=axes, **kwargs)
-
-    @pytest.mark.parametrize('tuple_ndim', [1, 3])
-    @pytest.mark.parametrize('mismatched',
-                             ['radius', 'mode', 'sigma', 'order'])
-    def test_gauss_axes_tuple_length_mismatch(self, tuple_ndim, mismatched):
+            ndimage.gaussian_filter(array, size, axes=axes)
```

Comment:
why was this line changed back from `filter_func`?
```suggestion
            filter_func(array, size, axes=axes)
```

Reply:
Oops I don't know.